css/styles.css

```diff
@@ -1,4 +1,219 @@
+}
+#dots:hover{
+    background-color:  rgb(240, 240, 240);
+    padding: 0px 16px 7px 10px;
+    margin: 0px -16px -7px -10px;
+    border-radius: 50%;
+    box-shadow: 0px 0px 1px 5px  rgb(240, 240, 240);
+}
+a:link{
+    color: currentColor;
+    text-decoration: none;
+}
+a:link:hover{
+    text-decoration:underline;
+}
+
+#logo{   
+    width:270px;   
+}
+.boxlogo{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-top: 154px;
+    
+
+}
+.search-bar{
+   width: 580px; 
+   height: 44px;
+   display: flex;
+   flex-direction:row;
+   align-items: center;
+   justify-content: center;
+   margin:auto; 
+   margin-top: 27px;
+   border-radius: 25px;
+   border: 1px solid rgb(225, 225, 226);
+}
+.search-bar:hover{
+    border-color: rgb(225, 225, 226);
+    box-shadow: 0px 0px 5px 1px rgb(225, 225, 226);
+}
+#glass{
+    width:18px;
+    margin-top: -3px;
+    margin-left: -28px;
+    padding-right: 22px;
+}
+#input-line{
+    border-color: transparent;
+    padding: 13px 117px 12px 117px;
+    margin-left: 10px;  
+}
+#keyboard{
+    width: 20px;
+    height: 11px;
+    margin-top: -1px;
+    margin-right: -2px;
+    cursor: pointer;
+    border-radius: 1px;
+}
+#microphone{
+    width: 15px;
+    height: 20px;
+    margin-top: -1px;
+    margin-left: 24px;
+    margin-right: 3px;
+    cursor: pointer;
+}
+#camera{
+    width: 20px;
+    margin-top: -2px;
+    margin-left: 20px;
+    margin-right: -25px;
+    cursor: pointer;
+}
+
+.search-buttons{
+    font-size: 14px;
+    color: rgb(32, 33, 36)C;;
+    border: 0px solid rgb(248, 249, 250); 
+    border-radius: 5px;
+    background-color: rgb(248, 249, 250);
+    padding: 10px 16px;
+    margin-top: 28px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+.search-buttons:hover{
+    box-shadow: 0px 0px 1px 1px  rgb(218, 220, 224);
+}
+#happy-link:link{
+        color:currentColor;
+        text-decoration: none;
+}
+#happy-link:visited{
+        color:currentColor;
+        text-decoration: none;
+}
+#langchange{
+ font-size: 13px;  
+ font-family:Arial, Helvetica, sans-serif;
+ margin-top: 29px;
+ margin-left: -10px; 
+}
+
+.country{
+    position: fixed;
+    top: auto;
+    bottom: 46px;
+    width: 100%;
+    padding: 15px;
+    padding-left: 28px;
+    margin-left: -7px;
+    text-align: left;
+    background-color:rgb(242, 242, 242); 
+    border: 1px solid rgb(242, 242, 242);
+    font-size: 15px;
+    font-family:Arial, Helvetica, sans-serif;
+    color:rgb(112, 117, 122)
+}
+.lowbar{
+    position: fixed;
+    bottom: 0px;
+    width: 100%;
+}
+.container{  
+    height: 45px;
+    padding-right: 13px;
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content:space-between;
+    margin-left: -7px;
+    background-color:rgb(242, 242, 242);
+    border: 1px solid rgb(218, 220, 224);
+    position:relative;
+    font-size: 14px;  
+    font-family:Arial, Helvetica, sans-serif;
+}     
+
+#leftbar{
```

Comment:
there are plenty of identical properties in leftbar and rightbar. think about crating a common class